Description: Update WSGI app creation to be compatible with Django 1.7
 This resolves the following error when trying to run Horizon:
 .
 AppRegistryNotReady: The translation infrastructure cannot be initialized
 before the apps registry is ready. Check that you don't make non-lazy gettext
 calls at import time.
 .
 and is backwards-compatible with our currently supported versions (back to
 1.4).
Author: Julie Pichon <jpichon@redhat.com>
Date: Thu, 16 Oct 2014 12:46:10 +0000 (+0100)
X-Git-Url: https://review.openstack.org/gitweb?p=openstack%2Fhorizon.git;a=commitdiff_plain;h=686af384c201148c3aebcfbf18b2527587d70ea4
Change-Id: I1e59f2bf6633a34cfe9230aa028f345e5fa70f94
Bug-Ubuntu: https://launchpad.net/bugs/1382023
Origin: Upstream, https://review.openstack.org/#/c/128911/
Last-Update: 2014-10-16

Index: horizon/openstack_dashboard/wsgi/django.wsgi
===================================================================
--- horizon.orig/openstack_dashboard/wsgi/django.wsgi
+++ horizon/openstack_dashboard/wsgi/django.wsgi
@@ -1,7 +1,7 @@
 import logging
 import os
 import sys
-import django.core.handlers.wsgi
+from django.core.wsgi import get_wsgi_application
 from django.conf import settings
 
 # Add this file path to sys.path in order to import settings
@@ -13,5 +13,4 @@ sys.path.append("/usr/share/openstack-da
 
 DEBUG = False
 
-application = django.core.handlers.wsgi.WSGIHandler()
-
+application = get_wsgi_application()
